### DISCUSSION TITLE
72% of the oldest open PRs are community PRs

### DISCUSSION THREAD

**Discussion by bhack** [keras-team/keras-cv#1184] [General]
https://github.com/keras-team/keras-cv/pulls?q=is%3Apr+sort%3Acreated-asc+is%3Aopen

72% of the oldest open PRs are "community" PRs.

How we could scale the review/approval process? Can we start to promote community maintainers?

Often going back to review a PR after months it means that the contribution could be lost. Generally external contributors could have a specific "dense" time-slot of available time that  often doesn't hold over months. Probably we have the same issue also in other ecosystem repos.

/ cc @tanzhenyu @qlzh727 @martin-gorner

**Comment by tanzhenyu** [comment]
Yep, this is very good point. We also have the idea of scaling this up through promoting community maintainers, now seems a good time (after 0.4 release) to do so. We need to polish the contribution guide dramatically to simplify the process for community maintainers

**Comment by bhack** [comment]
If you are interested, also if it was mainly forked from Pytorch, there are some Governance points under review at https://github.com/openxla/community/pull/28#discussion_r1054821796

**Comment by jbischof** [comment]
We've made a lot of progress on this now :)
